modules/core/src/v2/trading/trade.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @prettier
+ */
+export interface Trade {
+  id?: string;
+  externalId: string;
+  baseAccountId: string;
+  quoteAccountId: string;
+  timestamp: Date;
+  status: TradeStatus;
+  baseAmount: string;
+  quoteAmount: string;
+  baseCurrency: string;
```

Comment:
maybe make notes that these are 'ofc currencies?'

Reply:
:+1: